src/applications/burials/components/NoFormPage.jsx

```diff
@@ -0,0 +1,486 @@
+            Fill out an Application for Veterans Burial (VA Form 21P-530EZ).
+          </p>
+          <div>
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.vba.va.gov/pubs/forms/VBA-21P-530EZ-ARE.pdf"
+              pages={8}
+              text="Download VA form 21P-530EZ"
+            />
+            <p>
+              We’ve captured your intent to file date of{' '}
+              <strong>XX/XX/XXXX</strong>. You have 12 months from that date to
+              submit a claim.
+            </p>
+            <p className="vads-u-margin-bottom--4">
+              Mail your burial form to the pension management center:
+            </p>
+            <p className="va-address-block">
+              Department of Veterans Affairs <br />
+              Pension Intake Center
+              <br />
+              PO Box 5365
+              <br />
+              Janesville, WI 53547-5365
+              <br />
+            </p>
+            <article>
+              <va-on-this-page />
+              {renderFields.map(props => (
+                <CreateSummarySections
+                  {...props}
+                  formData={formData}
+                  key={props.id}
+                />
+              ))}
+            </article>
+            <p>
+              <strong>Note:</strong> According to federal law, there are
+              criminal penalties for withholding information on purpose or
+              providing information that you know is false. Penalties may
+              include a fine, imprisonment for up to 5 years, or both.
+              (Reference: 18 U.S.C. 1001)
+            </p>
+            <va-alert
+              background-only
+              class="vads-u-margin-bottom--1"
+              close-btn-aria-label="Close notification"
+              disable-analytics="false"
+              full-width="false"
+              status="info"
+              visible="true"
+            >
+              <p className="vads-u-margin-y--0">
+                <strong>
+                  Veterans Pension (VA Form 21P-527EZ) can not be currently
+                  completed online.
+                </strong>
+                <br />
+                We have saved your application so you can use it as a reference.
+                You will need to fill out a new form to apply by mail.
+              </p>
+            </va-alert>
+            <h2 className="vads-u-margin-bottom--0p5 vads-u-font-size--lg">
+              Need help?
+            </h2>
+            <hr className="vads-u-border-color--primary vads-u-margin-y--0 vads-u-border-bottom--2px" />
+            <p>
+              Call us at <va-link href="tel:800-827-1000" text="800-827-1000" />
+              . We’re here Monday through Friday, 8:00 a.m to 9:00 p.m ET. If
+              you have hearing loss, call TTY:{' '}
+              <va-link href="tel:711" text="711" />.
+            </p>
+          </div>
+        </>
+      ) : (
```

Comment:
This feels like a lot of logic to wrap in a ternary. Could we break this up into a couple functions returning the proper content instead?